spec/controllers/catalog_controller_spec.rb

```diff
@@ -54,6 +54,53 @@
       end
     end
   end
+  describe 'POST submit_cart_updates_form on item' do
+    before(:each) do
+      @equipment_model = FactoryGirl.create(:equipment_model)
+      put :add_to_cart, id: @equipment_model.id
```

Comment:
True, although technically shouldn't the `submit_cart_updates_form` method add an item to the cart with the specified quantity even if it wasn't originally there? In that case we wouldn't actually need to add an item to the cart beforehand since we don't care about the interface in controller specs.